symphony/content/content.systemauthors.php

```diff
@@ -668,6 +671,13 @@ public function __actionEdit()
                 || (Symphony::Author()->isDeveloper() && $this->_Author->isPrimaryAccount() === false);
 
         if (@array_key_exists('save', $_POST['action']) || @array_key_exists('done', $_POST['action'])) {
+
+            // Validate rights
+            if (!$canEdit && !$isOwner) {
+                $this->pageAlert(__('You are not allowed to edit this author.'), Alert::ERROR);
+                return;
+            }
+
```

Comment:
This can be remove. We will need to rebase that PR

Reply:
Not sure why you want to remove this? This is the only **the** place we verify that the author can edit another one